<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>citycollectionitem</class>
 <widget class="QWidget" name="citycollectionitem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>172</width>
    <height>102</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lbBackImage">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>12</y>
     <width>160</width>
     <height>90</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbDelete">
   <property name="geometry">
    <rect>
     <x>148</x>
     <y>0</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbCityName">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>28</y>
     <width>42</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbTmp">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>58</y>
     <width>39</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbTmpUnit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>59</y>
     <width>16</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>75</y>
     <width>30</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
